composer.json

```diff
@@ -25,7 +25,6 @@
         "ext-mbstring": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7 || ^6.5",
```

Comment:
It seems that you drop the PHPUnit and using your own test. I think it doesn't make sense.

Reply:
Why not? PHPUnit was only used for calling our own tests. The tests can easily be run without PHPUnit so I do not see what value was added by PHPUnit.

As I mentioned in a previous comment I removed PHPUnit because otherwise I could not test on PHP version >= 8. The benefit of removing PHPUnit is that now we can test on all possible versions of PHP. The tests are still the same.

Reply:
You can specify other PHPUnit versions so that it can test on PHP >= 8 versions.

Reply:
PHPUnit 7 has breaking changes, so we cannot use both version < 7 and >= 7.

Reply:
When I running the `PHPUnit 7.5.20` version with `php-7.4` and `php-8.0` for this `hjson-php`, it's worked successfully.

I don't get any breaking change.

Could you provide your building log about breaking change?

Reply:
My mistake, the breaking change is in PHPUnit 8, not 7. According to this page PHPUnit 8 is the first version that supports PHP version >= 8:

https://phpunit.de/supported-versions.html

With PHPUnit 8 I get this error:

    PHP Fatal error:  Declaration of HJSON\Tests\HJSONParserTest::setUp() must be compatible with 
    PHPUnit\Framework\TestCase::setUp(): void in /home/sikoaukt/hjson-php/tests/HJSONParserTest.php on line 10

You still haven't answered what value is added by PHPUnit?

Reply:
At this moment, perhaps we should not need PHPUnit. But we don't guarantee we should need that in the future.
If tests are growing in the future, we may notice that it's not good enough for own customized tests.
Another reason is: PHPUnit has some testing features and they can be good for developers to make tests easy and flexible.

Going back to the PHPUnit issue, this has two solutions can be resolved:

- Drop `php5.6` and `php-7.0` version supports, and let this package require `php-7.1` version at least. And the protected `setUp` method can change into `protected function setUp(): void`. And above issue can be resolved.
- Keeping PHPUnit versions for `^5`, `^6` and `^7`. And this can be supported for old and inactive PHP versions. We can  ignore `PHP 8` supports `PHPUnit 8` version issue.

Reply:
If needed we can easily start using PHPUnit again in the future, or some other more advanced tool that might exist in the future. But I doubt we will ever do that. The latest release of hjson-php was done almost three years ago, and I am the only member of the Hjson team left that still responds to issues and pull requests.

So I don't think we should use PHPUnit today. I prefer to have hjson-php support for all possible versions of PHP.

Reply:
If @dqsully agrees with this, just go for it.

Reply:
Unfortunately dqsully does not seem to be active in Hjson anymore, but I only need one approval. @peter279k can you approve?

Reply:
Sorry, last week was very busy for me and I'm just now getting around to everything. I think this all makes sense, if we don't need PHPUnit for our existing tests then we shouldn't have to require it at all for now.

Reply:
Thanks!